mailet/base/src/test/java/org/apache/mailet/base/test/FakeMail.java

```diff
@@ -214,57 +268,58 @@ public void setMessage(MimeMessage message) {
         }
     }
 
+    @Override
     public void setState(String state) {
         this.state = state;
     }
 
+    @Override
     public Serializable getAttribute(String name) {
         return attributes.get(name);
     }
 
+    @Override
     public Iterator<String> getAttributeNames() {
         return attributes.keySet().iterator();
     }
 
+    @Override
     public boolean hasAttributes() {
         return !attributes.isEmpty();
     }
 
+    @Override
     public Serializable removeAttribute(String name) {
         return attributes.remove(name);
 
     }
 
+    @Override
     public void removeAllAttributes() {
         attributes.clear();
     }
 
+    @Override
     public Serializable setAttribute(String name, Serializable object) {
-
         return attributes.put(name, object);
     }
 
+    @Override
     public long getMessageSize() throws MessagingException {
         return size;
     }
 
+    @Override
     public Date getLastUpdated() {
         return lastUpdated;
     }
 
+    @Override
     public void setLastUpdated(Date lastUpdated) {
         this.lastUpdated = lastUpdated;
     }
-    
+
     public void setMessageSize(long size) {
         this.size = size;
     }
-
-    public void setRemoteAddr(String remoteAddr) {
-        this.remoteAddr = remoteAddr;
-    }
```

Comment:
Sorry, i misread the link, it's not used now and it's also not part of the Mail interface.
We can remove it.